rust-mode.el

```diff
@@ -590,6 +641,28 @@ the desired identifiers), but does not match type annotations \"foo::<\"."
      (,(rust-re-grab (concat "#\\!?\\[" rust-re-ident "[^]]*\\]"))
       1 font-lock-preprocessor-face keep)
 
+     ;; Builtin formatting macros
+     (,(concat (rust-re-grab (concat (regexp-opt rust-builtin-formatting-macros) "!")) "[[:space:]]*[({[][[:space:]]*\"")
```

Comment:
That’s right. Today I learned that you can use those here, heh. I’ll update it accordingly.